t/mojo/dom.t

```diff
@@ -1040,28 +1040,28 @@ EOF
   is_deeply \@li, ['F'], 'found third last li element';
   @li = ();
   $dom->find('li:nth-child(1n+0)')->each(sub { push @li, shift->text });
-  is_deeply \@li, [qw(A B C D E F G)], 'found all li elements';
+  is_deeply \@li, [qw(A B C D E F G H)], 'found all li elements';
```

Comment:
This looks incorrect too. In Chrome i get `A C E F H I` for `li:nth-child(1n+0)`.

Reply:
Can't reproduce, at least with Firefox. I put
```html
<style>
    li:nth-child(1n+0) {
        outline: 1px solid green;
    }
</style>
<ul>
    <li>A</li>
    <li>B</li>
    <li>C</li>
    <li>D</li>
    <li>E</li>
    <li>F</li>
    <li>G</li>
    <li>H</li>
</ul>
```
in a local file and opened it in the browser. All list items got a green border. Also, running `document.querySelectorAll('li:nth-child(1n+0)').length` in the developer console gave `8`.

Reply:
Oh, I see what you did. You accidentally took the HTML code from line 1077, which is part of a different test. The HTML code you used has `<p>B</p>`, `<p>D</p>`, and `<p>G</p>`, which is why they don't show up with a `li` selector, and it has an additional `<li>I</li>` element. The correct HTML code for this test is in line 920 and doesn't contain any `I`.

Reply:
Oops, that's exactly what i did.